### ISSUE TITLE
to do list

### ISSUE THREAD

**Issue by artkapl** [OreoCassowary/datascience-workshop#1]
## Reading list

- [ ] string formatting in Python 3 - [String operations](https://docs.python.org/3.3/library/string.html)
- [ ] Theory behind IP adresses, [ASN](https://en.wikipedia.org/wiki/Autonomous_System_Number) (autonomous system number), Traceroute, announcing IP adresses
- [ ] [Unix time](https://en.wikipedia.org/wiki/Unix_time): date format "epoch" --> seconds from Jan 1, 1970. A common time stamp format in IT.

**Comment by sebix** [comment]
> string formatting in Python 3

If it's not necessary to convert from the "%s" style (i.e. you don't have complex substitutions/formatting), I'd not convert everything it.

**Comment by aaronkaplan** [comment]
Yes, but do you know python's 3.6's f-strings? That seems much nicer.
See: https://www.python.org/dev/peps/pep-0498/

**Comment by sebix** [comment]
> Yes, but do you know python's 3.6's f-strings? That seems much nicer.
The link goes to the 3.3 docs.

I don't think that 3.6 is widespread yet. I have it, but I'm using a rolling release distro.

**Comment by aaronkaplan** [comment]
> On 22 Jul 2017, at 17:43, Sebastian <notifications@github.com> wrote:
> 
> Yes, but do you know python's 3.6's f-strings? That seems much nicer.
> The link goes to the 3.3 docs.
> 
> I don't think that 3.6 is widespread yet. I have it, but I'm using a rolling release distort.

Sure :) But for the data science workshop, it does not matter :) it's just an experiment.
